### DISCUSSION TITLE
[Tables] Prefetching pagination

### DISCUSSION THREAD

**Discussion by Sicklou** [filamentphp/filament#8216] [General]
Hello filament people,

The V3 is a blast ! Really nice to use it.

I was wondering if it would be possible some day to use prefetching with the pagination of the tables.
I don't even know if it is possible at all to use the new navigate feature of Livewire elsewhere than full page component.

What do you guys think about this ?

**Comment by pboivin** [comment]
Hi @Sicklou. `wire:navigate` is coming, but there are issues with it at the moment. If you have some time, give it a test and see if you can contribute to this discussion: https://github.com/filamentphp/filament/pull/7197
